### DISCUSSION TITLE
关于程序运行时间

### DISCUSSION THREAD

**Discussion by Pragmatism0220** [Equim-chan/mjai-reviewer#82] [Q&A] [answered]
一场半庄战的麻将牌谱运行时间大概在多久比较正常呢？我本地运行一次大概要3分钟往上，有时候甚至会到将近5分钟。请问这正常吗？如果是的话，有没有什么参数、指令或着方法能够充分利用计算机运算资源、提升运行效率呢？谢谢！

**Comment by Equim-chan** [comment]
哪个 engine？

**Comment by Pragmatism0220** [comment]
> 

akochan

**Comment by Equim-chan** [comment]
这个很正常，没有什么办法。

**Comment by Pragmatism0220** [comment]
好吧谢谢。我还没有进行mortal的训练，所以想顺便请问一下，如果正确训练了mortal，假如akochan跑一次要3min，那mortal大概需要多长时间呢？

**Comment by Equim-chan** [comment, marked as answer]
Mortal 基本在 15 秒内。
